Sources/MapboxNavigation/NavigationCameraStateTransition.swift

```diff
@@ -14,6 +14,11 @@ public class NavigationCameraStateTransition: CameraStateTransition {
     var animatorBearing: BasicCameraAnimator?
     var animatorPitch: BasicCameraAnimator?
     
+    // The bearing change threshould to start animation of camera.
+    var bearingUpdateThreshold: CLLocationDirection = 5.0
```

Comment:
The default `CLLocationManager.headingFilter` is 1°. It doesn’t look like MapboxNavigation or MapboxMaps configures `CLLocationManager.headingFilter` anywhere, so we should probably expect to align to the default heading filter for consistency between the camera and the puck.

Reply:
I see. I'll use the default 1 for the value.